stats/opencensus/opencensus.go

```diff
@@ -103,15 +114,25 @@ func unaryInterceptor(ctx context.Context, method string, req, reply interface{}
 
 // streamInterceptor handles per RPC context management. It also handles per RPC
 // tracing and stats, and records the latency for the full RPC call.
-func streamInterceptor(ctx context.Context, desc *grpc.StreamDesc, cc *grpc.ClientConn, method string, streamer grpc.Streamer, opts ...grpc.CallOption) (grpc.ClientStream, error) {
+func (csh *clientStatsHandler) streamInterceptor(ctx context.Context, desc *grpc.StreamDesc, cc *grpc.ClientConn, method string, streamer grpc.Streamer, opts ...grpc.CallOption) (grpc.ClientStream, error) {
```

Comment:
Can you refactor this in order to share most of the code here with `unaryInterceptor`?

Reply:
Done.